rollup.config.js

```diff
@@ -10,7 +10,7 @@ const modulesDir = "entries"
 let modules = fs.readdirSync(modulesDir)
   .map((nameWithExtension) => nameWithExtension.split(".")[0])
 
-modules = ["layouts"]
+modules = ["global-alert"]
```

Comment:
This was my mistake. When rolling rollup, comment out this line. This will allow the system to run rollup for all the modules in the library.

During development/testing, you can specify which module you want to run though, since it takes a long time to rollup all the modules.